modules.d/90dmsquash-live-ntfs/module-setup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+command -v
+
+check() {
+    require_binaries ntfs-3g || return 1
+    return 0
```

Comment:
The final ```return 0``` replaced with ```return 255```. Of course that would make the module optional, but the LiveCD tools can pull it in, as it does with the dmsquash-live module